File: documentation-examples-scala/src/scala/dev/typr/foundationssc/docs/db2/StringTypesSingleByte.scala
package dev.typr.foundationssc.docs.db2
import dev.typr.foundationssc.*
import dev.typr.foundationssc.data.*

@SuppressWarnings(Array("unused"))
object StringTypesSingleByte:
  // start
  val charType: Db2Type[String] = Db2Types.char_
  val char10: Db2Type[String] = Db2Types.char_Of(10)
  val varcharType: Db2Type[String] = Db2Types.varchar
  val varchar255: Db2Type[String] = Db2Types.varcharOf(255)
  val clobType: Db2Type[String] = Db2Types.clob
  // stop

File: documentation-examples-java/src/java/dev/typr/foundations/docs/db2/StringTypesSingleByte.java
package dev.typr.foundations.docs.db2;

import dev.typr.foundations.Db2Type;
import dev.typr.foundations.Db2Types;

@SuppressWarnings("unused")
public class StringTypesSingleByte {
  // start
  Db2Type<String> charType = Db2Types.char_;
  Db2Type<String> char10 = Db2Types.char_(10);
  Db2Type<String> varcharType = Db2Types.varchar;
  Db2Type<String> varchar255 = Db2Types.varchar(255);
  Db2Type<String> clobType = Db2Types.clob;
  // stop
}
